exec/java-exec/src/test/java/org/apache/drill/exec/TestWindowFunctions.java

```diff
@@ -844,4 +844,37 @@ public void testStatisticalWindowFunctions() throws Exception {
         .build()
         .run();
   }
+
+  @Test // DRILL-2330
+  public void testNestedAggregates() throws Exception {
+
+    final String query = "select sum(min(l_extendedprice))" +
+            " over (partition by l_suppkey order by l_suppkey) as totprice" +
+            " from cp.`tpch/lineitem.parquet` where l_suppkey <= 10 group by l_suppkey order by 1 desc";
+
+    // Validate the plan
+    final String[] expectedPlan = {"Window.*partition \\{0\\} order by \\[0\\].*SUM\\(\\$1\\).*",
+            ".*SelectionVectorRemover.*",
```

Comment:
Ideally the regex used in the plan validation should only test for specific plan nodes.  In this case, is it sufficient to check for Window and the Aggregate and ignore the rest ?